fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt

```diff
@@ -7,65 +7,190 @@ package org.mozilla.fenix.tabstray
+                        modifier = Modifier.fillMaxHeight(),
+                        selectedContentColor = selectedColor,
+                        unselectedContentColor = inactiveColor,
+                    ) {
+                        Box {
+                            Icon(
+                                painter = painterResource(
+                                    id = mozilla.components.ui.tabcounter.R.drawable.mozac_ui_tabcounter_box,
+                                ),
+                                contentDescription = normalTabsContentDescription,
+                                modifier = Modifier.align(Alignment.Center),
+                            )
+
+                            Text(
+                                text = normalTabCountText,
+                                modifier = normalTabCountTextModifier.align(Alignment.Center),
+                                color = LocalContentColor.current,
+                                fontSize = with(LocalDensity.current) { 12.dp.toSp() },
+                                fontWeight = FontWeight.W700,
+                            )
+                        }
+                    }
+
+                    Tab(
+                        selected = selectedPage == Page.PrivateTabs,
+                        onClick = { onTabPageIndicatorClicked(Page.PrivateTabs) },
+                        modifier = Modifier.fillMaxHeight(),
+                        icon = {
+                            Icon(
+                                painter = painterResource(id = R.drawable.ic_private_browsing),
+                                contentDescription = stringResource(id = R.string.tabs_header_private_tabs_title),
+                            )
+                        },
+                        selectedContentColor = selectedColor,
+                        unselectedContentColor = inactiveColor,
+                    )
+
+                    Tab(
+                        selected = selectedPage == Page.SyncedTabs,
+                        onClick = { onTabPageIndicatorClicked(Page.SyncedTabs) },
+                        modifier = Modifier.fillMaxHeight(),
+                        icon = {
+                            Icon(
+                                painter = painterResource(id = R.drawable.ic_synced_tabs),
+                                contentDescription = stringResource(id = R.string.tabs_header_synced_tabs_title),
+                            )
+                        },
+                        selectedContentColor = selectedColor,
+                        unselectedContentColor = inactiveColor,
+                    )
+                }
             }
-        }
 
-        Box(modifier = Modifier.weight(1.0f)) {
-            PrimaryButton(text = "Synced tabs") {
-                onTabPageIndicatorClicked(Page.SyncedTabs)
+            Spacer(modifier = Modifier.weight(1.0f))
+
+            IconButton(
+                onClick = {},
```

Comment:
this will be handled in [Bug 1816517](https://bugzilla.mozilla.org/show_bug.cgi?id=1816517)